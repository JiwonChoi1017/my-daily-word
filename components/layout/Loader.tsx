import React from "react";
import classes from "../../styles/Loader.module.css";

/**
 * 読み込み中アニメーション.
 *
 * @returns {JSX.Element} 読み込み中アニメーション.
 */
const Loader = () => {
  return (
    <div className={classes.loader__wrap}>
      <span className={classes.loader} />
    </div>
  );
};

export default Loader;
