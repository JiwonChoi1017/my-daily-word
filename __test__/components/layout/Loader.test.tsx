import Loader from "@/components/layout/Loader";
import React from "react";
import { render } from "@testing-library/react";

describe("Loaderのテスト", () => {
  // Loaderコンポーネントをレンダリング
  test("Loader: スナップショットテスト", () => {
    const { container } = render(<Loader />);

    expect(container).toMatchSnapshot();
  });
});
