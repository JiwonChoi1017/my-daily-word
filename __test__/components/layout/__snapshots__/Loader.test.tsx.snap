// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loaderのテスト Loader: スナップショットテスト 1`] = `
<div>
  <div
    class="loader__wrap"
  >
    <span
      class="loader"
    />
  </div>
</div>
`;
